vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/contextmenu/ContextMenuWrapTest.java

```diff
@@ -200,8 +200,10 @@ void clickItem_byInvalidIndexes_throws() {
     @Test
     void clickItem_byNestedIndexNotUsableParent_throws() {
         // Hierarchical / NestedDisabled / Level3
+        menu_.open();
+
         Assertions.assertThrows(IllegalStateException.class,
-                () -> menu_.clickItem(7, 2, 0));
+                () -> menu_.clickItem(7, 3, 0));
```

Comment:
Should we for these actually have
```
IllegalStateException exception = Assertions.assertThrows(IllegalStateException.class,
                () -> menu_.clickItem(7, 3, 0));
Assertions.assertTrue(exception.getMessage().contains("THE EXPECTED MESSAGE"));
```
Just to make certain that the failure reason is correct as we do throw IllegalStateExceptions from many places.

Reply:
Good idea. I'll add the assertion

Reply:
Indeed there was another wrong test